Darkly.xcworkspace/xcshareddata/WorkspaceSettings.xcsettings

```diff
@@ -3,6 +3,6 @@
 <plist version="1.0">
 <dict>
 	<key>IDEWorkspaceSharedSettings_AutocreateContextsIfNeeded</key>
-	<true/>
+	<false/>
```

Comment:
Was this the fix for the schemes issue?

Reply:
in part. this prevents CocoaPods from creating schemes, which then cannot become public